zerver/openapi/zulip.yaml

```diff
@@ -12470,6 +12470,97 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+  /export/realm:
+    get:
+      operationId: get-realm-exports
+      summary: Get all the exports of the organization
+      tags: ["server_and_organizations"]
+      description: |
+        Fetch all the requested [data exports](/help/export-your-organization) of
+        the organization.
+
+        **Changes**: New in Zulip 2.1.
+      responses:
+        "200":
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccessBase"
+                  - additionalProperties: false
+                    properties:
+                      result: {}
+                      msg: {}
+                      ignored_parameters_unsupported: {}
+                      exports:
+                        type: array
+                        description: |
+                          An array of dictionaries where each dictionary contains
+                          data about a single organization export request.
+                        items:
+                          $ref: "#/components/schemas/RealmExport"
+                    example:
+                      {
+                        "exports":
+                          [
+                            {
+                              "acting_user_id": 11,
+                              "deleted_timestamp": null,
+                              "export_time": 1722243168.134179,
+                              "export_url": "http://example.zulipchat.com/user_avatars/exports/2/FprbwiF0c_sCN0O-rf-ryFtc/zulip-export-p6yuxc45.tar.gz",
+                              "failed_timestamp": null,
+                              "id": 323,
+                              "pending": false,
+                            },
+                          ],
+                        "msg": "",
+                        "result": "success",
+                      }
+    post:
+      operationId: export-realm
+      summary: Export data of the organization
```

Comment:
I'd revise to "Create a public data export of the organization" and the description below as well. Again we need to be clear this creates a public data export vs the other types of exports.

Reply:
Used `Create a public data export` to match the title in `rest-endpoints.md`.